src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
+
+find_package(Qt5 ${QT_MIN_VERSION} NO_MODULE REQUIRED
+	WebEngine
+	WebEngineWidgets
+)
+
+find_package(Radare2 1.4.0 REQUIRED)
+include_directories(${RADARE2_INCLUDE_DIRS})
+
+feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_INSTALL_PREFIX ${BUILD_DIR})
```

Comment:
It should be

iaito -> /usr/bin/iaito
{libs} x86_64 -> /usr/lib64
{libs} x86 -> /usr/lib
{other miscellaneous data files} -> /usr/share/iaito
{licence} -> /usr/share/licenses/iaito/
{themes} -> /usr/share/iaito/themes
{icons} -> /usr/share/icons/  or /usr/share/icons/iaito
logo -> /usr/share/iaito/iaito.png // this needs fix as commented in qmake project file